steps/container.py

```diff
@@ -233,3 +185,71 @@ def get_output(self, history=True):
     def remove_image(self, force=False):
         self.logging.info("Removing image %s" % self.image_id)
         d.remove_image(image=self.image_id, force=force)
+
+    def copy_file_to_container(self, src_file, dest_folder):
+        full_src_file=os.path.join(os.getcwd(), src_file)
```

Comment:
What if you specify absolute path? Unlikely, I know, but... :)

Reply:
I don't think we should. Since this is meant to run anywhere, e.g. CI won't like `/home/kabir/blah/blah/blah`. 

Of course there might be better ways to deal with this that I don't know yet :)

Reply:
I guess my point is that: why to not support it? The code will be even cleaner:

```python
if not os.path.isabs(src_file):
    src_file = os.path.abspath(os.path.join(os.getcwd(), src_file))
```

And that's it. You can use the `src_file` variable everywhere now, without introducing 2 new ones and adding support for absolute paths by the way.

Just one to make something clear: I'm not saying that absolute path should be used in feature files.

Reply:
Got it